tests/test-sources/plugins/by-name/tardis/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  empty = {
+    plugins.tardis.enable = true;
+  };
+
+  defaults = {
+    plugins.tardis = {
+      enable = true;
+
+      settings = {
+      };
+    };
+  };
+
```

Comment:
This is identical to your `empty` test. I don't think it's needed.

Historically, we would use the `defaults` test to explicitly define the "plugin defaults" that are documented by the `settingsOptions`. If you're keeping your explicitly declared settings options, then you'd copy the defaults you define there to this test. The idea being that this test validates those defaults are valid values for this plugin.

However, if we aren't declaring any `settings` options, there is nothing to validate and this test isn't needed.